client/src/panels/panel_declarations/finances/transfer_payments/tp_by_region.js

```diff
@@ -11,122 +11,274 @@ import {
   declarative_charts,
 } from "../../shared.js";
 import { Canada } from '../../../../charts/canada/index.js';
+import { SpinnerWrapper, TabbedContent } from '../../../../components/index.js';
+import { get_static_url, make_request } from '../../../../request_utils.js';
 
 const { std_years } = year_templates;
+
 const formatter = formats["compact2_raw"];
 
 const { text_maker, TM } = create_text_maker_component(text);
 const { provinces, provinces_with_article } = businessConstants;
 const { A11YTable } = declarative_charts;
 
-const calculate_common = (subject, table) => {
-  const level = subject.level;
-
-  if ( level === 'dept' && !_.has(table.depts , subject.id) ){
-    return false;
-  }
 
-  const data = std_years.map( year => table.prov_code(year, level === "gov" ? false : subject.id) );
+function loadPopulation(){
```

Comment:
I remember somewhere near the beginning that's what we had it as, but it wouldn't work since the arrow function made the function lexically bound and would crash the page

I can try it again to see if it will work now with the new updates though

Reply:
It seems to have worked; I'll wait for Circle CI to do a double check
Fixed for now 👍